src/julia/Integer.jl

```diff
@@ -42,6 +42,8 @@ one(::Integers{T}) where T <: Integer = T(1)
 
 is_unit(a::Integer) = a == 1 || a == -1
 
+is_nilpotent(a::Integer) = is_zero(a)
```

Comment:
Redundant?

Reply:
Not redundant: the point is that rings in `AbstractAlgebra` having integer or rational coefficients use `BigInt` or `Rational{BigInt}` to represent the values (rather than Oscar types).  I suppose a similar argument applies to the `is_unit` functions which already have specializations for `Integer` and `Rational`.

Reply:
They are redundant if the "generic" version allows `NCRingElement` instead of `NCRingElem` (which includes `Rational` and `Integer`). So yes, it should be removed and the other method adjusted.

Reply:
Removed the specialization for `Integer` after changing the generic implementation to `NCRingElement`